### PR TITLE
use seperate baseimage (main)

### PR DESCRIPTION
**Pull request by hydazz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-FROM ghcr.io/imagegenius/baseimage-ubuntu:lunar
+FROM ghcr.io/imagegenius/baseimage-immich:latest
```
Comment (martabal) [inline Dockerfile:3]: Wouldn't it better to keep the release name such as `ghcr.io/imagegenius/baseimage-immich:lunar`

File: Dockerfile [hunk 2]
```diff
@@ -26,64 +26,8 @@ RUN \
...
-    librsvg2-2 \
-    libspng0 \
-    libwebp7 \
-    libwebpdemux2 \
-    libwebpmux3 \
-    mesa-va-drivers \
-    nginx \
-    nodejs \
-    perl \
-    python3 \
```
Comment (martabal) [inline Dockerfile:83]: keep `python3`, `python3-pip` and `python3-venv`

File: Dockerfile [hunk 3]
```diff
@@ -164,39 +101,16 @@ RUN \
...
-    libspng-dev \
-    libtool \
-    libwebp-dev \
-    make \
-    meson \
-    ninja-build \
-    pkg-config \
-    python3-dev \
-    wget && \
+    python3-dev && \
```
Comment (martabal) [inline Dockerfile:105]: keep python3, python3-venv and remove `python3-dev`, python3-pip

**Review by hydazz — COMMENTED** [review]
.

**Pull request merged**